### PR TITLE
Release announcement for 2.25.1

### PR DESCRIPTION
**Pull request by jodygarnett**

### CODE REVIEW

File: _posts/2024-05-23-geoserver-2-25-1-released.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+- Vulnerability
+release: release_225
+version: 2.25.1
+jira_version: 16919
+--- 
+
+GeoServer [2.25.1](/release/2.25.1/) release is now available
+with downloads
+([bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.1/geoserver-2.25.1-bin.zip/download),
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.1/geoserver-2.25.1-war.zip/download),
```
Comment (petersmythe) [inline _posts/2024-05-23-geoserver-2-25-1-released.md:19]: I do not believe that Sourceforge latests have been updated to 2.25.1? https://sourceforge.net/projects/geoserver/files/latest/download

File: _posts/2024-05-23-geoserver-2-25-1-released.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+* [GEOS-11394](https://osgeo-org.atlassian.net/browse/GEOS-11394) OGC APIs cannot handle time extent when the source data type is java.sql.Date
+
+Community modules are shared as source code to encourage collaboration. If a topic being explored is of interest to you, please contact the module developer to offer assistance. 
+
+# About GeoServer 2.25 Series
+
+Additional information on GeoServer 2.25 series:
+
+* [GeoServer 2.25 User Manual](https://docs.geoserver.org/2.25.x/en/user/)
+* [GeoServer 2024 Roadmap Plannings]({% post_url 2024-01-03-roadmap %})
```
Comment (petersmythe) [inline _posts/2024-05-23-geoserver-2-25-1-released.md:112]: This hyperlink looks like it needs fixing

**Review by petersmythe — CHANGES_REQUESTED** [review]

File: _posts/2024-05-23-geoserver-2-25-1-released.md (hunk 2 above)
Comment (jodygarnett) [reply _posts/2024-05-23-geoserver-2-25-1-released.md:112]: that is a macro that gets filled in if you run locally

File: _posts/2024-05-23-geoserver-2-25-1-released.md (hunk 1 above)
Comment (jodygarnett) [reply _posts/2024-05-23-geoserver-2-25-1-released.md:19]: not sure how latest works?

Comment (petersmythe) [reply _posts/2024-05-23-geoserver-2-25-1-released.md:19]: Yes, but https://sourceforge.net/projects/geoserver/files/latest/download downloads geoserver-2.25.0-war.zip

Fixed now, as per

image

**Review by petersmythe — APPROVED** [review]

**Pull request merged**